packages/api/src/experimental/subscribe.ts

```diff
@@ -22,14 +22,12 @@ import type {
 import type { EXPERIMENTAL_ObjectSetListener } from "../public/unstable.js";
 import type { Experiment } from "./Experiment.js";
 
-type NonEmptyArray<T> = [T, ...T[]];
-
 type subscribeFn = <
   Q extends ObjectOrInterfaceDefinition,
   const P extends PropertyKeys<Q>,
 >(
   objectSet: ObjectSet<Q>,
-  properties: NonEmptyArray<P>,
+  properties: Array<P>,
```

Comment:
Probably should make this a `ReadOnlyArray<P>` too now that I think about it, so that people who have those in their redux can use it

Reply:
Will flup this with the other changes